src/broker/rusage.c

```diff
@@ -49,11 +49,11 @@ static void rusage_request_cb (flux_t *h, flux_msg_handler_t *mh,
             "nsignals", ru.ru_nsignals,
             "nvcsw", ru.ru_nvcsw,
             "nivcsw", ru.ru_nivcsw) < 0)
-        goto error;
+        flux_log_error (h, "%s: flux_respond_pack", __FUNCTION__);
```

Comment:
did you accidentally remove this `goto error`?  Seems not right.

Reply:
It didn't seem useful to handle a `flux_respond_pack()` error by calling `flux_respond_error()` since that is likely to fail also.  However, maybe I should do that in a separate cleanup commit.